pipeline/outputs/s3.md

```diff
@@ -119,6 +123,24 @@ Example:
 
 If you enable a single worker, you are enabling a dedicated thread for your S3 output. We recommend starting without workers, evaluating the performance, and then enabling a worker if needed. For most users, the plugin can provide sufficient throughput without workers.
 
+### Usage w/ MinIO
```

Comment:
nit: I like to be more formal in documentation writing, so no "w/"

Is that abbreviation completely universal? Would any user anywhere in the world who understood english enough to read this understand that? 

Another thought is that translation software probably deals better with full words not abbreviations.

Reply:
Fixed. Thank you for telling me that.